frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteAnnotator.java

```diff
@@ -121,23 +126,24 @@ private void processIndentationElement(PsiElement element, AnnotationHolder hold
     var wrongIndentChar = thisIndentationText.chars().filter(c -> c != indentationParameters.indentationCharacter).findFirst();
```

Comment:
Nit: use explicit `OptionalInt` since the type of `wrongIndentChar` isn't obvious